rpm_spec/subpackages/manageiq-appliance

```diff
@@ -82,6 +82,13 @@ if systemctl is-active --quiet evm-failover-monitor; then
   systemctl restart evm-failover-monitor
 fi
 
+# Disable rsyslog duplicating systemd-journal output
+# This will comment out the multi-line module load from /etc/rsyslog.conf
+sed -i '/^module(load="imjournal"/, /StateFile="imjournal.state")/ s|^|#|' %{_sysconfdir}/rsyslog.conf
```

Comment:
If you aren't a `sed` expert (which I am not), this means "starting at this regex" (`/^module(load="imjournal"/`) until this regex (`/StateFile="imjournal.state")/`), substitute the start of the line with `#` (`s|^|#|`)